themes/bootstrap3/templates/RecordDriver/SolrOverdrive/status-full.phtml

```diff
@@ -28,20 +25,19 @@ foreach ($ids as $id) {
         $current['error'] = $this->transEsc('status_unknown_message');
         $current['full_status'] = "";
     } elseif ($loginForAvail || $avail->code == 'od_code_login_for_avail') {
-        $current['full_status']
-            = "<p class='alert-info'>Login for availability</p>";
+        $current['full_status'] = "<p class='alert-info'>"
+            . $this->transEsc('od_code_login_for_avail') . "</p>";
     } else {
-        $current['full_status']
-            = "<p><div class='copies'><strong>Total Copies</strong> "
-            . $avail->copiesOwned . "</div>
-                        <div class='avail'><strong>Available</strong> "
-            . $avail->copiesAvailable . "</div>
-                        <div class='holds'><strong>Holds</strong> "
+        $current['full_status'] = "<p><div class='copies'><strong>"
+            . $this->transEsc('od_avail_total') . "</strong> "
+            . $avail->copiesOwned . "</div><div class='avail'><strong>"
```

Comment:
Is there any chance that $avail contains something else than numbers? Maybe escape these to be sure?

Reply:
It should always be numeric, but it never hurts to escape just in case. Done now!